webpack.config.js

```diff
@@ -87,6 +87,17 @@ module.exports = function(env) {
         {
           test: /script\/.*\.js$/,
           use: [{loader: 'script-loader'}]
+        },
+        {
+          test: /\.(woff|woff2|eot|ttf|otf)$/,
+          use: [
+            {
+              loader: 'file-loader',
+              options: {
+                outputPath: 'assets/font',
+              },
+            }
+          ]
```

Comment:
Is this actually necessary, or a relic of conversion?

Reply:
I think it may be necessary if we embed fonts anywhere else on the site.